lib/freight/apt.sh

```diff
@@ -207,6 +207,12 @@ EOF
 
     } >"$DISTCACHE/Release"
 
+    # Pinentry may fail on GnuPG 2.x without GPG_TTY
+    #
+    # https://github.com/freight-team/freight/issues/72
+    #
+    GPG_TTY=$(tty)
```

Comment:
I'm unsure if this works that way generally, e.g. in a cronjob, could you test that?

Reply:
Based on my testing this works properly even from cron. Not having it and cron reports the same failure as when testing manually. When this is present, freight-cache is happy, and cron is happy.

That said Travis seemed unhappy with not this being exported, so I will update this PR accordingly.